### DISCUSSION TITLE
voting expansion

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#8518] [Enhancement]
I really want to see a feature:
![image](https://github.com/Zrips/CMI/assets/141560260/a249f894-b909-46be-9ed5-e3f0e070d233)
this is the only option for voteparty, I would like there to be an event TotalVotes, which does the same but not for one player but for all players and then resets when the number is reached. Also with a placeholder for your TAB and scoreboard

**Comment by ghost** [comment]
any update?

**Comment by CloudeLecaw** [comment]
We made a little workarround for this, this with a gobal Placeholders that counts up every vote

Its checked by CMI Scheduler each hour.
- When the Number 50 (oh higher) is reached when the check is made then it executed the 50rewards commands for all players online and sets a seound global placeholder to 1 (the command is not exeuted aggain because it checks if the secound placeholer is 0)
- When the Number 100 (or higher) is reached when the check is made then it executed the 100rewards commands for all players online and sets a third global placeholder to 1 (the command is not exeuted aggain because it checks if the third placeholer is 0)
- When the Number 200 (or higher) is reached when the check is made then it executed the 200rewards commands for all players online and sets a fouth global placeholder to 1 (the command is not exeuted aggain because it checks if the fourth placeholer is 0)
- At the End of this little scheduler - When the Number 200 (or higher) is reached then all four placeholders are reseted to 0

Well the little limitation to this is the check is not live for each vote. its just done once each hour (time depending on the scheduler defined time)
